src/propulsion/motor_builder.hpp

```diff
@@ -12,20 +13,14 @@
 namespace propulsion {
 
 /**
- * @brief A static class which only exists to build a correct Motor object
+ * @brief A class which only exists to build a correct Motor object
  */
-class MotorBuilder {
+class MotorBuilder final : public AbstractMotorBuilder {
  public:
-  /// Constructor is deleted, because it should not be instantiated
-  MotorBuilder() = delete;
+  MotorBuilder() = default;
+  ~MotorBuilder() = default;
```

Comment:
Why is it not necessary to have a 'virtual' destructor here?

Reply:
because this class is final and can't be inherited.